build-system/babel-plugins/babel-plugin-amp-dev-assert-transformer/index.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Is a MemberExpression "thenable", `foo().then(() => {});`
```

Comment:
Isn't this just asking if the expression is a promise or not? What else would be "thenable" besides a promise?